templates/prometheus.go

```diff
@@ -29,9 +30,8 @@ var PrometheusSpecTemplate = promv1.PrometheusSpec{
 		ListenLocal:            true,
 		Resources:              defaults.MonitoringResources["prometheus"],
 		Containers: []corev1.Container{{
-			Name: "kube-rbac-proxy",
-			// Tech-debt to include kube-rbac-proxy image from environment
-			Image: "gcr.io/kubebuilder/kube-rbac-proxy:v0.13.0",
+			Name:  "kube-rbac-proxy",
+			Image: os.Getenv("KUBE_RBAC_PROXY_IMAGE"),
```

Comment:
I don't remember if we ensure anywhere that this env is set?

Reply:
We do ensure this env is set here 
https://github.com/ezio-auditore/ocs-operator/blob/6defc783a9f091af1d9ab6662c84b7bc2df8b98f/controllers/storagecluster/storagecluster_controller.go#L46

Reply:
ah ok.